### PR TITLE
run make depsupdate

### PR DESCRIPTION
**Pull request by hobbitronics**

### CODE REVIEW

File: installed-versions.json [hunk 1]
```diff
@@ -1,14 +1,20 @@
 {
+  "version": "6.0.0",
```
Comment (briskt) [inline installed-versions.json:2]: I don't recall this being here, or if we have a process to make sure it shows the correct version at release time. Maybe it doesn't matter much unless it shows in Sentry logs or AWS logs.

File: installed-versions.json [hunk 2]
```diff
@@ -1,14 +1,20 @@
...
   "name": "idp-profile-ui",
   "dependencies": {
     "@babel/polyfill": {
       "version": "7.12.1",
       "resolved": "https://registry.npmjs.org/@babel/polyfill/-/polyfill-7.12.1.tgz",
       "overridden": false
     },
+    "@sentry/vue": {
```
Comment (briskt) [inline installed-versions.json:10]: I don't understand why vue is just now showing up as a dependency.

File: installed-versions.json [hunk 3]
```diff
@@ -1,14 +1,20 @@
...
+      "version": "7.119.0",
+      "resolved": "https://registry.npmjs.org/@sentry/vue/-/vue-7.119.0.tgz",
+      "overridden": false
+    },
     "@simplewebauthn/browser": {
-      "version": "4.1.0",
-      "resolved": "https://registry.npmjs.org/@simplewebauthn/browser/-/browser-4.1.0.tgz",
+      "version": "9.0.1",
```
Comment (briskt) [inline installed-versions.json:16]: big jump! How long has it been since the last update?

**Review by briskt — APPROVED** [review]

File: installed-versions.json (hunk 1 above)
Comment (hobbitronics) [reply installed-versions.json:2]: It has been in the package.json file, but didn't ever get updated until we installed sentry. I suppose we could add it to a checklist, maybe in the PR template (which doesn't exist yet)

File: installed-versions.json (hunk 2 above)
Comment (hobbitronics) [reply installed-versions.json:10]: I forgot to run `make depsupdate` after installing sentry and probably other times dependencies were changed. Probably another thing to add to a checklist in a PR template.

File: installed-versions.json (hunk 3 above)
Comment (hobbitronics) [reply installed-versions.json:16]: Another time I forgot to run `make depsupdate`. See https://github.com/silinternational/idp-profile-ui/pull/169 for details, but the short story is the repo has ui and server, and most of the changes happen in the server code, so not many breaking changes in the ui code.

**Pull request merged**